elephant/spectral.py

```diff
@@ -523,6 +529,543 @@ def multitaper_psd(signal, n_segments=1, len_segment=None,
     return freqs, psd
 
 
+def multitaper_cross_spectrum(signals, fs=1., nw=4, num_tapers=None,
+                              peak_resolution=None, return_onesided=True,
+                              attach_units=True):
+    """
+    Estimates the cross spectrum of a given 'neo.AnalogSignal' using the
```

Comment:
Classes of objects mentioned in the text (main docstring or any text explanation in the sections below) should be embraced by backticks, so that they are shown in italics on the HTML documentation page (e.g., `neo.AnalogSignal`). The exception is in the parameter list, where they are written without any ticks (see line 553 below)

Reply:
Addressed in https://github.com/NeuralEnsemble/elephant/pull/525/commits/5ddd042c88783fdc4203819a36146c746510599d